chef/cookbooks/crowbar/files/default/crowbar.service

```diff
@@ -1,15 +1,29 @@
 [Unit]
-Description=Crowbar Daemon
+Description=Crowbar
 After=network.target syslog.target remote-fs.target chef-server.service
 
 [Service]
 Type=simple
 User=crowbar
 Group=crowbar
-PIDFile=/var/run/crowbar/crowbar.pid
+KillMode=control-group
+TimeoutSec=300
+TimeoutStopSec=10
+
+WorkingDirectory=/opt/dell/crowbar_framework
+
+Environment="CROWBAR_ENV=production"
+Environment="CROWBAR_THREADS=2"
+Environment="CROWBAR_WORKERS=4"
+Environment="CROWBAR_LISTEN=127.0.0.1"
+Environment="CROWBAR_PORT=80"
+Environment="CROWBAR_ENV_FILE=/etc/sysconfig/crowbar"
```

Comment:
So what is actually using this file (or this environment variable)? I don't see it ever used?